exercises/pig-latin/.meta/src/reference/java/PigLatinTranslator.java

```diff
@@ -47,16 +45,15 @@ private String translateWord(String word) {
         return word.substring(1) + word.toCharArray()[0] + AY;
     }
 
-    private static boolean hasYAfterConsonantCluster(String word) {
-        for(int i = 0; i < word.length() - 1; i++){
-            if(word.substring(i, i + 1).matches(VOWELS_REGEX)){
-                return false;
-            }else if((i != 0) && word.substring(i, i + 1).equals("y")){
-            	yLocation = i;
-                return true;
+    private static int getLocationOfYAfterConsonantCluster(String word) {
+        for (int i = 0; i < word.length() - 1; i++) {
+            if (word.substring(i, i + 1).matches(VOWELS_REGEX)) {
+                return -1;
+            }else if ((i != 0) && word.substring(i, i + 1).equals("y")) {
```

Comment:
There should be a white space before `else` 🙂 Also you could use `word.charAt(i)` in both the `if` and `else` statements (e.g.  `word.charAt(i) == 'y'`) instead of using `substring`, that might make it a bit more obvious what you're doing 🙂